src/services/pictours/index.jsx

```
@@ -316,6 +317,7 @@ export const PicTourPreviewer = ({ image, isOpen, onClose, isTop = false }) => {
         commentText,
         setCommentText,
         totalComments,
+        title: image?.name,
```

ESLint

[error] 'image.name' is missing in props validation

(react/prop-types)